modules/removing-cso-operator.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * security/pod-vulnerability-scan.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="uninstalling-container-security-operator_{context}"]
+= Uninstalling the {rhq-cso}
+
+To uninstall the Container Security Operator, you must uninstall the Operator and delete the `imagemanifestvulns.secscan.quay.redhat.com` custom resource definition (CRD). Without removing the CRD, image vulnerabilities are still reported on the {product-title} *Overview* page.
+
+.Procedure
+
+. On the {product-title} web console, click *Operators* -> *Installed Operators*.
+
+. Click the menu {kebab} of the Container Security Operator.
+
+. Click *Uninstall Operator*. Confirm your decision by clicking *Uninstall* in the popup window.
+
+. Remove the `imagemanifestvulns.secscan.quay.redhat.com` custom resource definition by entering the following command:
++
+[source,terminal]
+----
+$ oc delete customresourcedefinition imagemanifestvulns.secscan.quay.redhat.com
```

Comment:
Suggest add the response in the doc to help customer know and confirm the expected behavior
```
oc delete customresourcedefinition imagemanifestvulns.secscan.quay.redhat.com
customresourcedefinition.apiextensions.k8s.io "imagemanifestvulns.secscan.quay.redhat.com" deleted
```